docs/_providers/axfrddns.md

```diff
@@ -0,0 +1,107 @@
+---
+name: AXFRDDNS
+title: AXFR+DDNS Provider
+layout: default
+jsId: AXFRDDNS
+---
+# AXFR+DDNS Provider
+
+This provider is able to work with any authoritative DNS server accepting AXFR requests (RFC5936) and Dynamic Update (RFC2136).
+
+It has been tested with [BIND](https://www.isc.org/bind/), [Knot](https://www.knot-dns.cz/), and [Yadifa](https://www.yadifa.eu/home).
+
+## Configuration
+
+### Authentication
+
+The AXFR+DDNS provider might work without anything in `creds.json` if the primary master of the zone accepts transfers and updates without TSIG authentication. But for non widely-open server, the authentication keys should be provided in `creds.json`.
+
+{% highlight json %}
+{
+    "axfrddns": {
+        "transfer-key": "hmac-sha256:transfer-key-id:Base64EncodedSecret=",
+        "update-key": "hmac-sha256:update-key-id:AnotherSecret="
+    }
+}
+{% endhighlight %}
+
+The `transfer-key` will be used to authenticate AXFR request, and the `update-key` will be used to authenticate the Dynamic Updates. Both keys are optional, and you could provide only one them.
+
+If distinct zones requires distinct keys, you might instantiate the provider multiple times:
+
+{% highlight javascript %}
+var AXFRDDNS_A = NewDnsProvider('axfrddns-a', 'AXFRDDNS'}
+var AXFRDDNS_B = NewDnsProvider('axfrddns-b', 'AXFRDDNS'}
+{% endhighlight %}
+
+And update `creds.json` accordingly:
+
+{% highlight json %}
+{
+    "axfrddns-a": {
+        "transfer-key": "hmac-sha256:transfer-key-id:Base64EncodedSecret=",
+        "update-key": "hmac-sha256:update-key-id:AnotherSecret="
+    },
+    "axfrddns-b": {
+        "transfer-key": "hmac-sha512:transfer-key-id-B:SmallSecret=",
+        "update-key": "hmac-sha512:update-key-id-B:YetAnotherSecret="
+    }
+}
+{% endhighlight %}
+
```

Comment:
Sure. Would you like a `named.conf` that matches the example with distinct keys for distinct domains ? Or just a simple `named.conf` that accept authenticated tranfer and update ?

I can provide both but for the former, rather than a single server requiring different credentials, I think the use case is more likely when domains are hosted on distinct servers. Each servers being managed by distinct provider and hence the distinct credential.

Reply:
I think the audience is a person that has run BIND for a while, who wants to add updates via DNSControl.  I think seeing what needs to be added to one domain is sufficient. They can figure out multiple domains and multiple servers if the understand the simple case very well.

Reply:
Ok, thanks. Done